engine/class_modules/sc_monk.cpp

```diff
@@ -9366,7 +9366,7 @@ std::string monk_t::default_flask() const
       break;
     case MONK_MISTWEAVER:
       if ( true_level > 110 )
-        return "endless_fathoms";
+        return "greater_flask_of_endless_fathoms";
```

Comment:
⚠️ This conflicts with PR simulationcraft/simc#4942 commit 8e29110851d501980edaf96dcf988a3d52522965

However it is possibly a more appropriate fix. Either way the merge conflict resolution should be trivial.